backend/src/routes/subscriber.ts

```diff
@@ -0,0 +1,110 @@
+import { PrismaClient } from "@prisma/client/edge";
+import { withAccelerate } from "@prisma/extension-accelerate";
+import { Hono } from "hono";
+import { verify } from "hono/jwt";
+
+export const subscriberRouter = new Hono<{
+  Bindings: {
+    DATABASE_URL: string;
+    JWT_SECRET: string;
+  };
+  Variables: {
+    userId: string;
+  };
+}>();
+
+subscriberRouter.use("/*", async (c, next) => {
+  try {
+    const header = c.req.header("authorization") || "";
+    const token = header && header.split(" ")[1];
+    const user = await verify(token, c.env.JWT_SECRET);
+    if (user) {
+      c.set("userId", user.id);
+      return next();
+    } else {
+      c.status(403);
+      return c.json({ error: "Unauthorized" });
+    }
+  } catch (e) {
+    c.status(403);
+    return c.json({
+      error: "Credentials failed",
+    });
+  }
+});
+
+subscriberRouter.post("/subscribe", async (c) => {
+  const prisma = new PrismaClient({
+    datasourceUrl: c.env.DATABASE_URL,
+  }).$extends(withAccelerate());
+
+  const { userId, subscriberId } = await c.req.json();
+  
+  try {
+    await prisma.subscriber.create({
+      data: {
+        userId,
+        subscriberId,
+      },
+    });
```

Comment:
There needs to be a validation check in place.

What if the Aadesh has already subscribed to Lokesh ? A duplicate entry would be created if we don't check first.